jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/enricher/QuarkusHealthCheckEnricher.java

```diff
@@ -52,23 +56,25 @@ public QuarkusHealthCheckEnricher(JKubeEnricherContext buildContext) {
 
     @Override
     protected Probe getReadinessProbe() {
-        return discoverQuarkusHealthCheck(asInteger(getConfig(Config.READINESS_INTIAL_DELAY, "5")));
+        return discoverQuarkusHealthCheck(asInteger(getConfig(Config.READINESS_INTIAL_DELAY, "5")),
+            READY_SUBPATH);
     }
 
     @Override
     protected Probe getLivenessProbe() {
-        return discoverQuarkusHealthCheck(asInteger(getConfig(Config.LIVENESS_INITIAL_DELAY, "10")));
+        return discoverQuarkusHealthCheck(asInteger(getConfig(Config.LIVENESS_INITIAL_DELAY, "10")),
+            LIVE_SUBPATH);
     }
 
-    private Probe discoverQuarkusHealthCheck(int initialDelay) {
+    private Probe discoverQuarkusHealthCheck(int initialDelay, String subPath) {
         if (!getContext().hasDependency("io.quarkus", "quarkus-smallrye-health")) {
             return null;
         }
 
         return new ProbeBuilder()
             .withNewHttpGet()
               .withNewPort(asInteger(getConfig(Config.PORT)))
-              .withPath(getConfig(Config.PATH))
+              .withPath(Paths.get("/", getConfig(Config.HEALTH_PATH), subPath).toString())
```

Comment:
I think that if you invoke this in Windows, you'll get a back-slash as default separator. Which is not what you expect for a URL path.

Reply:
Probably String#join is something that will work better in this case.

Reply:
I think is not the case. This is an URL path so the "/" should be fine

Reply:
I haven't checked but [JavaDocs](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Paths.html#get-java.lang.String-java.lang.String...-) suggest that this method will use the default FileSystem [name-separator](https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileSystem.html#getSeparator()), which for windows is  `\`.

Anyway, don't worry. I'll check this and change it if necessary

Reply:
![image](https://user-images.githubusercontent.com/488391/108734596-ea50ab00-752f-11eb-9e04-199506410bc4.png)

As suspected and indicated by the JavaDoc, running the above line in Windows produces a wrong URL with backslashes instead of common slashes.

Reply:
Anyway, I'll fix this in a subsequent commit. The rest seems fine. Thx!